### ISSUE TITLE
Cant install with yarn

### ISSUE THREAD

**Issue by danilobuerger** [mixmaxhq/redfour#19]
```
error redfour@1.0.4: The engine "yarn" is incompatible with this module. Expected version "YARN NO LONGER USED - use npm instead.".
error Found incompatible module
```

Why is this incompatible to yarn?

**Comment by skeggse** [comment]
Fixed in `v1.0.5`. Sorry about that!

**Comment by danilobuerger** [comment]
Thanks for the quick response!
